components/apimgt/org.wso2.carbon.apimgt.rest.api.store/README.md

```diff
@@ -198,6 +199,7 @@ beans.xml
             <ref bean="PreAuthenticationInterceptor"/>
             <ref bean="AuthenticationInterceptor"/>
             <ref bean="PostAuthenticationInterceptor"/>
+            <bean id="SubscriberRegistrationInterceptor">
```

Comment:
Any difference between `<ref bean=` and `<bean id=`?

Reply:
This is a typo in README. Fixed it.